lib/linter-registry.js

```diff
@@ -66,10 +66,10 @@ export default class LinterRegistry {
     const filePath = editor.getPath()
 
     if (
-      (onChange && !this.lintOnChange) ||                                          // Lint-on-change mismatch
-      !filePath ||                                                                 // Not saved anywhere yet
-      isPathIgnored(editor.getPath(), this.ignoreGlob, this.ignoreVCS) ||          // Ignored by VCS or Glob
-      (!this.lintPreviewTabs && editor.hasTerminatedPendingState === false)        // Ignore Preview tabs
+      (onChange && !this.lintOnChange) ||                                                       // Lint-on-change mismatch
+      !filePath ||                                                                              // Not saved anywhere yet
+      isPathIgnored(editor.getPath(), this.ignoreGlob, this.ignoreVCS) ||                       // Ignored by VCS or Glob
+      (!this.lintPreviewTabs && atom.workspace.getActivePane().getPendingItem() === editor)     // Ignore Preview tabs
```

Comment:
Looks like the pending item stuff was added in 1.6.0, with some refactoring in 1.7.0. So our minimum version of v1.4.0 doesn't even know pending items _exist_ 😛.